test/e2e/testdata/backend-tls-settings.yaml

```diff
@@ -25,42 +30,68 @@ spec:
+            - type: File
+              file:
+                path: /dev/stdout
+---
+# openssl req -x509 -sha256 -nodes -days 36500 -newkey rsa:2048 -subj '/O=example Inc./CN=example.com' -keyout ca.key -out ca.crt
 apiVersion: v1
 data:
   ca.crt: |
     -----BEGIN CERTIFICATE-----
-    MIIDQzCCAiugAwIBAgIBATANBgkqhkiG9w0BAQsFADBCMRMwEQYDVQQKEwpFbnZv
-    eVByb3h5MRAwDgYDVQQLEwdHYXRld2F5MRkwFwYDVQQDExBFbnZveSBHYXRld2F5
-    IENBMCAXDTI0MDMxMDE1MzIxN1oYDzIxMjQwMzEwMTYzMjE3WjBCMRMwEQYDVQQK
-    EwpFbnZveVByb3h5MRAwDgYDVQQLEwdHYXRld2F5MRkwFwYDVQQDExBFbnZveSBH
-    YXRld2F5IENBMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA7ZFmGB4e
-    m1KdGEohAZBfqydAEGLDHJ1YyfHWdd+vBAevdW64bZx3pggJOtgCnePuFd02rDQS
-    dlsJlX/6mFtoQilo6wvxDSJRfaTDbtfTjw+7k8yfd/Jsmh0RWG+UeyI7Na9sXAz7
-    b57mpxsCoNowzeK5ETiOGGNWPcjENJkSnBarz5muN00xIZWBU+yN5PLJNxZvxpZJ
-    Ol/SSI8sno0e0PxAmp3fe7QaXiZj/TAGJPGuTJkUxrHqyZGJtYUxsS8A0dT1zBjj
-    izA5Dp+b5yzYo23Hh7BgpbZ7X4gsDThFuwCD6fHyepuv2zHPqvSsdqg2hAhDp91R
-    zrn7a9GxG2VSIwIDAQABo0IwQDAOBgNVHQ8BAf8EBAMCAQYwDwYDVR0TAQH/BAUw
-    AwEB/zAdBgNVHQ4EFgQUUpP1aZ1M2KIuPPWrNPDV2c5CngowDQYJKoZIhvcNAQEL
-    BQADggEBAGSEkAVz+Z0qS4FmA0q4SCpIIq64bsdEjiUzev7pK1LEK0/Y28QBPixV
-    cUXfax18VPR9pls1JgXto9qY+C0hnRZic6611QTJlWK1p6dinQ/eDdYCBC+nv5xx
-    ssASwmplIxMvj3S1qF6dr7sMI2ZVD5HElTWdO19UBLyhiKKZW2KxDsYj+5NRwGFe
-    G+JuDgq7njUM8mdyYk0NehefdBUEUUCQtnwUtW95/429XwqQROuRDteGT9kjD+Y5
-    ea5mW4mfqLeuGJXZs9bdWjKKdLQPrn9IshPysWqz2Hz8dQ1f7N9/g8UWVSjd4cyx
-    S5EAolzVv0yB7wHCWCgfG/ckdOTUNnE=
+    MIIDPTCCAiWgAwIBAgIUCPaRrg+0wCmznOMf9HxtYXPc+rowDQYJKoZIhvcNAQEL
+    BQAwLTEVMBMGA1UECgwMZXhhbXBsZSBJbmMuMRQwEgYDVQQDDAtleGFtcGxlLmNv
+    bTAgFw0yNTA1MTMwMTI0MDRaGA8yMTI1MDQxOTAxMjQwNFowLTEVMBMGA1UECgwM
+    ZXhhbXBsZSBJbmMuMRQwEgYDVQQDDAtleGFtcGxlLmNvbTCCASIwDQYJKoZIhvcN
+    AQEBBQADggEPADCCAQoCggEBAO7E7Gtx+tUS5UujhJ/DdSqsqQQ391WnrQbI1i9c
+    tiYSj8p8/eq1dAw4u3gG5426UaVph8LG5GLYCGT8XPmeWQMZncGvf9TeqlmBz/WM
+    NJ6Ow7dh6pWfGuZ7FHkv+l7L7pd/SFd8z0yuLBeTkE3weZ0jcnoThpm7Unu5ULdV
+    /ZE0rnAG7cjqiTALj/QE1gF0VkXw6D9fT5SamKYasYT0ZQND5tVOlLoqXjYU1fJF
+    Gyqs7hI24c6KeB1G6giWvwiUSzHBmCL0seCDDGeIjhtO7YubRZqNHaE1cw3FFlb7
+    FwXbff8XVb85Mxy6Jwdq27sCy4iMUSBVgbRhLRb9Cp+iUBMCAwEAAaNTMFEwHQYD
+    VR0OBBYEFAhZJV5u/3sXGacaUkuP5zIFZYGiMB8GA1UdIwQYMBaAFAhZJV5u/3sX
+    GacaUkuP5zIFZYGiMA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEB
+    AJJmuyGwjP9KG1uwhfJ5jq3b+vUUZW29HnFzEfd50GAk++W1WBTyxpWXCX0VW4Va
+    /QcJJiROXoKUqGtqPHYKx8UQ/mzrmlRGMgKwYReoMLU6ukydox9AUDVQFnOQZm26
+    QTCwXb288715UeK+c+TdmpkFkTPh3F67HDn0q9mtoT9g9BckJRxoHaez1LfxbytW
+    nNBx3Rpk1Ps79EfjtKL1BJIvG/N/ar9Jey2CR+tlVKwrrPCGG1ObcGvHooQWOOCq
+    7jD99R8PXZV5VDyP8qoHPYYyqkaQH4nQtVm288vFNQRF+/vUUQsNKl/JQcTG2UCc
+    IwP0azf9fJ+Xxx19TOMO3FA=
     -----END CERTIFICATE-----
 kind: ConfigMap
 metadata:
-  name: backend-tls-certificate
+  name: ca-certificate
   namespace: gateway-conformance-infra
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: tls-backend
+  name: tls-backend-policy
```

Comment:
use a new service name instead of overriding the one with same name in manifests.